projects/assets-library/assets/styles/_color-palette.scss

```diff
@@ -45,7 +45,7 @@ $green-9: #03110a;
 $blue-1: #f0f6ff;
 $blue-2: #b8d3ff;
 $blue-3: #70a7ff;
-$blue-4: #2478ff;
+$blue-4: #0666ff;
```

Comment:
Do we want to change blue-4 everywhere or should we just use a different blue like blue-3 or blue-5 for whatever instance you are trying to fix?

Reply:
If we change this we will presumably have to look at all the pages we have already changed again, because the color contrast between the new `blue-4` and other colors changes now.

Reply:
If we change this we will presumably have to look at all the pages we have already changed again, because the color contrast between the new blue-4 and other colors changes now.

Reply:
I'm increasing the color value to just 7%. For the user will be really similar and we can keep the logic of hover, already applied in links but this passes the accessibility check.

![image](https://github.com/hypertrace/hypertrace-ui/assets/79482271/54e86461-2c66-44b4-90ae-e328334cd7b5)

Reply:
Okay, I'm cool with changing the color, but @Exac is right that we will need to recheck areas where this might cause regressions.

@palbizu please ensure no regressions before merging. @Exac if you know any specific places Pato should check, please let him know.

I'll approve, but please check before merging.

Reply:
I don't know anywhere in particular that we should check. The point being that the tool we use to check for accessibility on all the other pages did so when `blue-4` was `2478ff`. Now that we are changing one of our palette colors, we will have to go back and run the accessibility tool for every page that we have already merged, in order to be sure.

It shouldn't take too long, we can just look at the PRs that have been merged and look at the accessibility tool for each page to confirm we didn't change anything. It can even be done as a final pass once we have completed all these accessibility stories.

I support this change, just want to make sure that we don't get to the end of all these accessibility changes and realize that we inadvertently introduced different contrast problems.

Reply:
Yes I understand that could be an issue, I took a look at several components and they are working fine. There are many uses for links with this color this change helps a lot to accessibility